web/package.json

```diff
@@ -1,13 +1,13 @@
 {
 	"name": "@hashgraph/stablecoin-dapp",
-	"version": "1.24.0",
+	"version": "1.25.0",
 	"files": [
 		"build/"
 	],
 	"scripts": {
 		"start": "react-app-rewired start",
 		"build": "react-app-rewired build",
-		"test": "react-app-rewired test --maxWorkers=50% --watchAll=false",
+		"test": "react-app-rewired test --maxWorkers=25% --watchAll=false --runInBand --cache=false",
```

Comment:
max workers and runInBand seem to be incompatible with each other.

From the checks: 
[Both --runInBand and --maxWorkers were specified, but these two options do not make sense together. Which is it?](https://github.com/hashgraph/stablecoin-studio/actions/runs/10813475080/job/29997591581?pr=1180#:~:text=Both%20%2D%2DrunInBand%20and%20%2D%2DmaxWorkers%20were%20specified%2C%20but%20these%20two%20options%20do%20not%20make%20sense%20together.%20Which%20is%20it%3F)

I recommend removing the `--runInBand` argument here.

Reply:
Thanks! It was a copy paste error 👀